integration-tests/cucumber-tests-platform-common/src/test/java/org/opensmartgridplatform/cucumber/platform/common/glue/database/WsCoreNotificationDatabase.java

```diff
@@ -9,31 +9,36 @@
  */
 package org.opensmartgridplatform.cucumber.platform.common.glue.database;
 
-import org.opensmartgridplatform.cucumber.platform.common.glue.steps.database.ws.CoreApplicationKeyConfigurationRepository;
-import org.opensmartgridplatform.cucumber.platform.common.glue.steps.database.ws.CoreNotificationWebServiceConfigurationRepository;
-import org.opensmartgridplatform.cucumber.platform.common.glue.steps.database.ws.CoreResponseDataRepository;
-import org.opensmartgridplatform.cucumber.platform.common.glue.steps.database.ws.CoreResponseUrlDataRepository;
-import org.springframework.beans.factory.annotation.Value;
+import org.opensmartgridplatform.adapter.ws.domain.repositories.ApplicationKeyConfigurationRepository;
+import org.opensmartgridplatform.adapter.ws.domain.repositories.NotificationWebServiceConfigurationRepository;
+import org.opensmartgridplatform.adapter.ws.domain.repositories.ResponseDataRepository;
+import org.opensmartgridplatform.adapter.ws.domain.repositories.ResponseUrlDataRepository;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
 @Component
 public class WsCoreNotificationDatabase extends WsNotificationDatabase {
 
   public WsCoreNotificationDatabase(
-      final CoreResponseDataRepository responseDataRepository,
-      final CoreResponseUrlDataRepository responseUrlDataRepository,
-      final CoreNotificationWebServiceConfigurationRepository
-          notificationWebServiceConfigurationRepository,
-      final CoreApplicationKeyConfigurationRepository applicationKeyConfigurationRepository,
-      @Value("${web.service.notification.port}") final int webServiceNotificationPort,
-      @Value("${web.service.notification.context}") final String webServiceNotificationContext) {
+      @Qualifier("wsCoreResponseDataRepository")
+          final ResponseDataRepository responseDataRepository,
```

Comment:
Is het niet fraaier om hier niet de Qualifier te gebruiken maar de naam van het argument anders te maken.... `          final ResponseDataRepository wsCoreResponseDataRepository,`

Reply:
Please stick to English when working or commenting on pull requests in the OSGP repositories.

Reply:
I changed the comments to English

Reply:
Personally I'd like to keep the variable names somewhat shorter. I have renamed the repository classes to get rid of the qualifiers here.